spec/system/claims/support/support_users/add_a_support_user_spec.rb

```diff
@@ -0,0 +1,88 @@
+require "rails_helper"
+
+RSpec.describe "Add a support user", type: :system do
+  include ActiveJob::TestHelper
+
+  around do |example|
+    perform_enqueued_jobs { example.run }
+  end
+
+  let!(:support_user) { create(:claims_support_user, :colin) }
+
+  scenario "Add a support user" do
+    when_i_sign_in_as_a_support_user(support_user)
+    and_i_visit_the_support_users_page
+    and_i_click_on_add_a_support_user
+    then_i_fill_in_the_support_user_form(email_address: "john.doe@education.gov.uk")
+    i_am_redirected_to_check_the_support_user_details
+    then_i_click_on_add_user
+    i_see_the_support_user_has_been_added
+    and_an_email_is_sent_to_the_support_user(email_address: "john.doe@education.gov.uk")
+  end
+
+  scenario "Attempt to add a support user without an @education.gov.uk email address" do
+    when_i_sign_in_as_a_support_user(support_user)
+    and_i_visit_the_support_users_page
+    and_i_click_on_add_a_support_user
+    then_i_fill_in_the_support_user_form(email_address: "john.doe@example.com")
+    then_i_see_an_error
+  end
+
+  private
```

Comment:
I think there are two scenarios missing here. One "Attempt to add an already-existing support user". 

And the other (which can be handled implicitly in the first scenario), is navigating back to the form using the change or the back button, making a change and then continuing. That would test the functionality, but it also tests that all the expected elements (the back button, the change links) are on the page. This ensures we'll catch it if we are make changes to the UI and accidentally delete things. (been known to happen...)